examples/aura.yaml

```diff
@@ -0,0 +1,39 @@
+participants:
```

Comment:
Lets update the configuration in the readme to talk about the `cl_client_type` being set to null behavior! Sorry for the confusion we go with `cl_client_type` instead of `cl_client_image`!